platform/Cargo.toml

```diff
@@ -43,6 +43,8 @@ cw-multi-test = "0.16.1"
 cosmos-sdk-proto = { version = "0.18", default-features = false }
 
 # Other
+serde-json-wasm = { version = "1", default-features = false }
+
```

Comment:
I missed adding a comment on this before, why do you go directly with `serde`? We use `sdk::cosmwasm_std::from_json` and `sdk::cosmwasm_std::to_json_*`.